omod/src/main/resources/apps/iss_patient_dashboard_app.json

```diff
@@ -28,9 +28,10 @@
     "description": "Patient Medical Conditions",
     "order": 1,
     "config": {
-      "widget": "obsacrossencounters",
+      "widget": "latestobsforconceptlist",
       "icon": "icon-user-md",
       "label": "MEDICAL CONDITIONS",
+      "nLatestObs": 5,
```

Comment:
@reagan-meant increase this to 10, so that all conditions can be returned

Reply:
Done